src/nft/components/nftExploreBanner/NftExploreBanner.tsx

```diff
@@ -0,0 +1,129 @@
+import { Trans } from '@lingui/macro'
+import { Box } from 'nft/components/Box'
+import { subhead } from 'nft/css/common.css'
+import { X } from 'react-feather'
+import { useNavigate } from 'react-router-dom'
+import { useShowNftPromoBanner } from 'state/user/hooks'
+import styled, { useTheme } from 'styled-components/macro'
+import { StyledInternalLink } from 'theme'
+import { Z_INDEX } from 'theme/zIndex'
+
+import nftPromoImage1 from '../nftExploreBanner/nftArt1.png'
+import nftPromoImage2 from '../nftExploreBanner/nftArt2.png'
+import nftPromoImage3 from '../nftExploreBanner/nftArt3.png'
+
+function getRandom(list: any[]) {
+  return list[Math.floor(Math.random() * list.length)]
+}
+const randomizedNftImage = getRandom([nftPromoImage1, nftPromoImage2, nftPromoImage3])
+
+const PopupContainer = styled.div<{ show: boolean }>`
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  box-shadow: ${({ theme }) => theme.deepShadow};
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
+  border-radius: 12px;
+  cursor: pointer;
+  color: ${({ theme }) => theme.textPrimary};
+  height: 100px;
+  display: ${({ show }) => (show ? 'flex' : 'none')};
+  flex-direction: column;
+  bottom: 48px;
+  position: fixed;
+  right: clamp(0px, 1vw, 16px);
+  text-decoration: none;
+  width: ${Math.min(391, window.innerWidth - 5)}px;
+  z-index: ${Z_INDEX.sticky};
+  transition: ${({
+    theme: {
+      transition: { duration, timing },
+    },
+  }) => `${duration.slow} opacity ${timing.in}`};
+  :hover {
+    border: double 1px transparent;
+    border-radius: 12px;
+    background-image: ${({ theme }) =>
```

Comment:
it might be the screenshots in Jira are out of date but the gradient looks reversed here. In the pictures it goes light -> dark but in the build I'm getting dark -> light